apps/backend/src/libs/modules/token/base-token.ts

```diff
@@ -0,0 +1,33 @@
+import { JWTPayload, jwtVerify, JWTVerifyResult, SignJWT } from "jose";
+
+type ConstructorProperties = {
+	algorithm: string;
+	expirationTime: string;
+	secret: Uint8Array;
+};
+
+class BaseToken<T extends JWTPayload> {
+	private algorithm: string;
+	private expirationTime: string;
+	private secret: Uint8Array;
+
+	constructor({ algorithm, expirationTime, secret }: ConstructorProperties) {
+		this.secret = secret;
+		this.algorithm = algorithm;
+		this.expirationTime = expirationTime;
+	}
+
+	public async createToken(payload: T): Promise<string> {
+		return await new SignJWT(payload)
+			.setProtectedHeader({ alg: this.algorithm })
```

Comment:
Lets bound type of algorithm to "HS256" and expirationTime to "24hr".
We will have 
```
type Algorithm = "HS256"
type ExpirationTime = 24hr""
```
We can use these types here and in apps/backend/src/libs/modules/config/base-config.module.ts

Reply:
Should we support other algorithms or only HS256? Same question goes for expiration time

Reply:
I do not think that we will do it now but it will be useful to do to know what to expect from .env variables. I think it will be convenient

Reply:
What do you mean by "We can use these types here and in apps/backend/src/libs/modules/config/base-config.module.ts"? Do you want me to use these type in format fields of convict configuration object? Or do you want me to use them somewhere else?